packages/ui/src/ui/store/reducers/groups/table.ts

```diff
@@ -0,0 +1,105 @@
+import {GROUPS_TABLE, GROUPS_TABLE_DATA_FIELDS} from '../../../constants/groups';
```

Comment:
I manually renamed these files via `git mv`, but once again - git decided that it's a new file.